plugins/chat/lib/onebox/templates/discourse_chat_thread.mustache

```diff
@@ -0,0 +1,29 @@
+<aside class="onebox chat-onebox">
```

Comment:
Bit strange choice here for aside.

Reply:
It's part of the default onebox styling :man_shrugging: 
https://github.com/discourse/discourse/blob/7019dbcfe123626c279e3f3f5733683eaec4008f/app/assets/stylesheets/common/base/onebox.scss#L94-L315